InvalidProductException.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Lekcia11_cvicenie
+{
+    internal class InvalidProductException : Exception
+    {
+        public string ProductName { get; }
+        public decimal Price { get; }
+        public InvalidProductException(string massage, string productName, decimal price) : base(massage)
```

Comment:
super pouziji jak noveho zapisu pro property, o kterem jsme mluvili a take umyslnem vynechani setu!! :)

Taky jsi super pouzil to, ze si ve sve vyjimce muzes vytvorit vlastni property, diky kterym pak zlepsis napr. vypis chyby